MMM-TeslaFi.js

```diff
@@ -256,6 +265,34 @@ Module.register("MMM-TeslaFi", {
           }
           break;
 
+        //static map - GoogleAPI needed and is only used if vehicle is not at tagged locations
+        case "map":
+          if (this.config.googleMapApiKey !== "") {
+            if (!this.isExcluded(t.location) || t.carState === "Driving") {
+              table += `
+              <tr>
+                <td class="icon ${
+                  t.carState !== "Driving" ? "dim" : ""
+                }" colspan="3">
```

Comment:
Could you explain why you want this dimmed while the car is not driving? To me, it would make sense to show the location all the time?

Reply:
It does show the location all the time.
If the car isn't driving - that map isn't updating. 
It was a little distracting to me to have one VERY bright area of the screen (showing the map) compared to the rest of the display.
I only really wanted to see the map if it was updating. 
That's all.

Reply:
Thanks - lets go with this then! We'll see how it looks when merged.